src/font.rs

```diff
@@ -64,7 +64,7 @@ pub enum FormatVersion {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct MetaInfo {
-    pub creator: String,
+    pub creator: Option<String>,
```

Comment:
I think this is right, but in line with the other changes you've been making we could also just treat the empty string as none. (I prefer this way, though, it's more explicit and more "correct".

Reply:
Mh, an empty string would still land in the serialized stream unless I stop serde; and the spec doesn't say that the field must be non-empty :thinking: